include/packetgraph/nic.h

```diff
@@ -95,4 +95,31 @@ int pg_nic_get_mtu(struct pg_brick *brick, uint16_t *mtu,
  */
 void pg_nic_get_mac(struct pg_brick *nic, struct ether_addr *addr);
 
+#define PG_NIC_RX_OFFLOAD_VLAN_STRIP  0x00000001
```

Comment:
I would add a #ifndef PG_NIC_RX_OFFLOAD_VLAN_STRIP before defining everything to manage the case where dpdk has been include before packetgraph/nic.h